resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/kubernetes/integrationtest/docker/SparkDockerImageBuilder.scala

```diff
@@ -19,14 +19,20 @@ package org.apache.spark.deploy.kubernetes.integrationtest.docker
 import java.net.URI
 import java.nio.file.Paths
 
-import com.fasterxml.jackson.jaxrs.json.JacksonJaxbJsonProvider
-import com.spotify.docker.client.{DefaultDockerClient, DockerCertificates}
+import com.spotify.docker.client.{DefaultDockerClient, DockerCertificates, LoggingBuildHandler}
 import org.apache.http.client.utils.URIBuilder
 import org.scalatest.concurrent.{Eventually, PatienceConfiguration}
 import org.scalatest.time.{Minutes, Seconds, Span}
 
 private[spark] class SparkDockerImageBuilder(private val dockerEnv: Map[String, String]) {
 
+  private val DOCKER_BUILD_PATH = Paths.get("target", "docker")
+  // Dockerfile paths must be relative to the build path.
+  private val DRIVER_V1_DOCKER_FILE = "dockerfiles/driver/Dockerfile"
+  private val DRIVER_V2_DOCKER_FILE = "dockerfiles/driver-v2/Dockerfile"
+  private val EXECUTOR_DOCKER_FILE = "dockerfiles/executor/Dockerfile"
```

Comment:
are you planning to do the executor V2 change in a separate PR?  This one's already plenty big enough, I lose attention halfway through the 2k+ lines